requirements/optionals.txt

```diff
@@ -6,4 +6,8 @@ django-polymorphic>=2.1
 django-rest-polymorphic>=0.1.8
 django-oauth-toolkit>=1.2.0
 djangorestframework-camel-case>=1.1.2
-django-filter>=2.3.0
\ No newline at end of file
+django-filter>=2.3.0
+pymongo>=3.11.0
```

Comment:
`pymongo` is not required as it is a dependency of `mongoengine`

`mongomock` will likely also come in handy for testing